hw2/05-form.js

```diff
@@ -1 +1,44 @@
-// Add your code here
+const labelStyle = document.querySelectorAll("label");
+const form = document.querySelector("form");
+const checkBoxStyle = document.querySelectorAll("span");
+const inputStyle = document.querySelectorAll("input");
+
+function handleSubmitForm(event) {
+    event.preventDefault();
+
+    const name = document.getElementById("userName").value;
+    const email = document.getElementById("userEmail").value;
+    const regStatus = document.getElementById("regStatus").value;
+    const coursesTaken = [];
+    const checkedBoxes = document.getElementsByName("coursesTaken");
+    const addComment = document.getElementById("comment").value;
```

Comment:
Since form fields are wrapped in <form>, you can query form directly for its elements and values. Which saves a lot of variable declarations.

Reply:
no changes made to this.